sys/include/embUnit/AssertImpl.h

```diff
@@ -49,7 +49,7 @@ void assertImplementationCStr(const char *expected,const char *actual, long line
         __typeof__(expected_) ____expected__ = expected_; \
         __typeof__(actual_) ____actual__ = actual_; \
         if (stdimpl_strcmp(____expected__, ____actual__) != 0) { \
-            assertImplementationCStr(____expected__, ____actual__, __LINE__, __FILE__); \
+            assertImplementationCStr(____expected__, ____actual__, __LINE__, RIOT_FILE_RELATIVE); \
```

Comment:
Should we really change this here? When someone acts upon the great idea to upgrade to another version of embUnit, this change my get lost?